index.html

```diff
@@ -273,132 +273,141 @@ <h2>About Me</h2>
-                />
-                <h3>skills</h3>
-              </div>
-              <div class="stack-skills">
-                <ul>
-                  <li>Codekit</li>
-                  <li>Github</li>
-                  <li>Codepen</li>
-                  <li>Gitlab</li>
-                  <li>Terminal</li>
-                </ul>
-              </div>
-              </li>
-             </ul>
+                  <div class="stack-heading">
+                    <img
+                      src="./assets/icons/Rectangle 55.svg"
+                      alt="language-icon"
+                    />
+                    <h3>Language</h3>
+                  </div>
+                  <div class="stack-skills">
+                    <ul>
+                      <li>Javascript</li>
+                      <li>Ruby</li>
+                      <li>Html</li>
+                      <li>Css</li>
+                    </ul>
+                  </div>
+                </li>
+                <li>
+                  <div class="stack-heading stack-skills_styled">
+                    <img
+                      src="./assets/icons/Rectangle 56.svg"
+                      alt="language-icon"
+                    />
+                    <h3>Frameworks</h3>
+                  </div>
+                  <div class="stack-skills">
+                    <ul>
+                      <li>Bootstrap</li>
+                      <li>Ruby On Rails</li>
+                      <li>RsPec</li>
+                      <li>CapyBara</li>
+                      <li>Selenium</li>
+                    </ul>
+                  </div>
+                </li>
+                <li>
+                  <div class="stack-heading stack-skills_styled">
+                    <img
+                      src="./assets/icons/Rectangle 57.svg"
+                      alt="language-icon"
+                    />
+                    <h3>skills</h3>
+                  </div>
+                  <div class="stack-skills">
+                    <ul>
+                      <li>Codekit</li>
+                      <li>Github</li>
+                      <li>Codepen</li>
+                      <li>Gitlab</li>
+                      <li>Terminal</li>
+                    </ul>
+                  </div>
+                </li>
+              </ul>
             </div>
           </div>
         </section>
 
         <section class="contact-info">
           <h3>
-            I'm always interested in hearing about new projects, so if you'd
-            like to chat please get in touch.
+           If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
           </h3>
-          <form action="mailto:ekiugbotega@gmail.com" method=”POST” enctype=”text/plain” class="contact-form">
+          <form
+            action="https://formspree.io/f/mqkoqlor"
+            method="”POST”"
+            class="contact-form"
+          >
```

Comment:
- [ ] Kindly ensure you correct your typing error on your `method` as it blocks your form to not send a message.
```HTML
         <form
            action="https://formspree.io/f/mqkoqlor"
            method="POST"
            class="contact-form"
          >
           ....
          </form>
```